assets/javascript/script.js

```diff
@@ -178,6 +200,17 @@ $(document).ready(function () {
             let results = response;
             console.log(results);
 
+<<<<<<< HEAD
```

Comment:
conflicts are not a big deal you guys I have had a conflict every time I have pulled practically . I had to resolve conflicts before I could pull Neri's code.  But anyway, I will pull the other two merges and then push this again...